### PR TITLE
documentation for gaugegroup and sp2n

### PR DESCRIPTION
**Pull request by LupoA**

### CODE REVIEW

File: documentation/manual.rst [hunk 1]
```diff
@@ -2778,46 +2778,82 @@ and there are associated reconstruction routines for assembling four spinors fro
...
 
-The most important external methods are::
+  namespace GroupName {
+  class SU {};
+  class Sp {};
+  }
 
-  static void printGenerators(void) ;
-  template <class cplx>  static void generator(int lieIndex, iSUnMatrix<cplx> &ta) ;
+A simpler interface is achieved by instantiating the GaugeGroup class with a specific group::
```
Comment (chillenzer) [inline documentation/manual.rst]: "instantiating" --> "aliasing"

File: documentation/manual.rst [hunk 2]
```diff
@@ -2778,46 +2778,82 @@ and there are associated reconstruction routines for assembling four spinors fro
...
 
-  static void GaussianFundamentalLieAlgebraMatrix(GridParallelRNG &pRNG,
-                                                  LatticeMatrix &out,
-                                                  Real scale = 1.0) ;
-  static void GaugeTransform( GaugeField &Umu, GaugeMat &g)
-  static void RandomGaugeTransform(GridParallelRNG &pRNG, GaugeField &Umu, GaugeMat &g);
+  template <int ncolour>
+  using Sp = GaugeGroup<ncolour, GroupName::Sp>;
+  
+Specific instantiations are then defined::
```
Comment (chillenzer) [inline documentation/manual.rst]: again "aliases"

**Review by chillenzer — CHANGES_REQUESTED** [review]

**Comment by LupoA** [comment]
On it

**Comment by LupoA** [comment]
done!

**Pull request merged**